ruby/chat_pub.rb

```diff
@@ -0,0 +1,3 @@
+channel = ably.channels.get('room:tesla-fans')
+channel.presence.enter_client('client', 'metadata')
```

Comment:
This should simply be `channel.presence.enter` like the JavaScript snippet (which I've just fixed in https://github.com/ably/homepage-snippets/pull/11)